test/Vester.t.sol

```diff
@@ -1,12 +1,111 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.13;
 
+import "semitransferable-token/Token.sol";
 import "forge-std/Test.sol";
+import "src/Vester.sol";
 
 contract VesterTest is Test {
-    function setUp() public {}
+    uint256 constant public SUPPLY = 1e9 ether;
+    address public $owner = address(this);
+    Token public token;
+    address public $token;
+    Vester public vester;
+    address public $vester;
+    address public $recipient = address(1);
 
-    function testExample() public {
-        assertTrue(true);
+    // Vesting data
+    address usr = $recipient;
+    uint256 tot = 100 ether;
+    uint256 bgn;
+    uint256 tau = 10;
+    uint256 eta = 0;
+    address mgr = $owner;
+    bool res = false;
+    bool bls = false;
+
+    function setUp() public {
+        token = new Token("Token", "TKN", 18, $owner);
+        $token = address(token);
+        vester = new Vester($owner, $token);
+        $vester = address(vester);
+
+        token.setRoleCapability(0, Token.transfer.selector, true);
+        token.setRoleCapability(0, Token.transferFrom.selector, true);
+        token.setRoleCapability(1, Token.mint.selector, true);
+
+        token.mint($owner, SUPPLY); // Mint 1B tokens to owner.
+        token.approve($vester, SUPPLY); // Allow vester to spend tokens.
+
+        // TWENTY_YEARS is needed here to avoid underflow on DssVest contract.
+        bgn = block.timestamp + vester.TWENTY_YEARS() + 100;
```

Comment:
I don't understand the `TWENTY_YEARS` here

Reply:
Updated to make it clearer!